File: POC_Angular/src/app/author/book.model.ts
import { DATE_PIPE_DEFAULT_TIMEZONE } from "@angular/common";

export class Book{
    id:number=0;
    title:string='';
    author:string='';
    authorId:number= 0;
    publisher:string='';
    category:string ='';
    content:string='';
    activeStatus:string='';
    blocked:boolean=false;
    price:number=0;
    image?:FormData;
    publishedDate:Date = new Date;
}

File: POC_API/POC_API/DbModels/bookModel.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace POC_API.DbModels
{
    public class bookModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public int? AuthorId { get; set; }
        public string Publisher { get; set; }
        public string Category { get; set; }
        public string Content { get; set; }
        public string ActiveStatus { get; set; }
        public bool? Blocked { get; set; }
        public int? Price { get; set; }
        public IFormFile ImagePath { get; set; }
        public DateTime? PublishedDate { get; set; }
    }
}
